helix-term/src/commands.rs

```diff
@@ -515,6 +523,90 @@ impl PartialEq for MappableCommand {
     }
 }
 
+#[derive(Clone)]
+pub struct FallbackCommand {
+    pub name: &'static str,
+    pub fun: fn(cx: &mut Context, event: KeyEvent),
+    pub with_prompt: MappableCommand,
```

Comment:
Why do we need `with_prompt`?

Reply:
For use in `reverse_map`, so that the binding for the fallback command shows up for its prompting version in the command palette. I suppose it could be left out, but I think the binding should really show up in the palette. It could also be just the name instead, since that's what ultimately ends up getting used.

At one point, I tried to come up with a way to have just a prompt message that'd allow for the "generation" of an appropriately structured function, but I couldn't figure that out. Doing that might require a third `MappableCommand` variant, and I'm not sure that's super worth looking into.

Reply:
Is it possible not have it behaves like a command without needing any prompt?